### PR TITLE
Pia 1580

### PR DESCRIPTION
**Pull request by zladzeyka**
* Add method without a networking call which takes a list of schemes as a parameter([String]) and returns a boolean value.
  * Add delegate method should handle errors internally which allow users to handle errors by their own

### CODE REVIEW

File: Example/Example Swift/CaptureSDK/ComponentAPICoordinator.swift [hunk 1]
```diff
@@ -630,6 +630,17 @@ extension ComponentAPICoordinator {
 // MARK: GiniPayBusinessDelegate
 
 extension ComponentAPICoordinator: GiniPayBusinessDelegate {
+    func shouldHandleErrorInternally(error: GiniPayBusinessError) -> Bool {
+        switch error {
+        case .noInstalledApps:
+            // shows own error
+            print("Didn't find any installed banking apps")
```
Comment (a-szotyori) [inline Example/Example Swift/CaptureSDK/ComponentAPICoordinator.swift:637]: This print can go easily unnoticed. Could you show an alert view instead with the same message?

File: GiniPayBusiness/Classes/Core/GiniPayBusiness.swift [hunk 2]
```diff
@@ -13,7 +13,7 @@ import GiniPayApiLib
  Makes use of callback for handling payment request creation.
  
  */
-@objc public protocol GiniPayBusinessDelegate {
+public protocol GiniPayBusinessDelegate: AnyObject {
```
Comment (a-szotyori) [inline GiniPayBusiness/Classes/Core/GiniPayBusiness.swift:16]: 👍

File: GiniPayBusiness/Classes/Core/GiniPayBusiness.swift [hunk 3]
```diff
@@ -115,6 +124,24 @@ public struct DataForReview {
         self.getInstalledBankingApps(completion: completion)
     }
     
+    /**
+     Checks if there is any banking app which can support Gini Pay functionaly installed.
```
Comment (a-szotyori) [inline GiniPayBusiness/Classes/Core/GiniPayBusiness.swift:128]: Typo: `functionality`

File: GiniPayBusiness/Classes/Core/GiniPayBusiness.swift [hunk 4]
```diff
@@ -115,6 +124,24 @@ public struct DataForReview {
...
+     
+     - parameter appSchemes: A list of [LSApplicationQueriesSchemes] added in Info.plist. Scheme format: ginipay-bank://
+     Return boolean value.
```
Comment (a-szotyori) [inline GiniPayBusiness/Classes/Core/GiniPayBusiness.swift:131]: You could use the `- returns:` [section](https://nshipster.com/swift-documentation/#parameters-return-values) for that.

Generally I would recommend paying attention to the format of the documentation comments. For example removing the empty line at line 132 would be a start. 😉 

Here's a great guide I found (of course it's NSHipster 😄): https://nshipster.com/swift-documentation

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Well done, just minor things.

File: Example/Example Swift/CaptureSDK/ComponentAPICoordinator.swift (hunk 1 above)
Comment (zladzeyka) [reply Example/Example Swift/CaptureSDK/ComponentAPICoordinator.swift:637]: Sure the thing is that  alert view is shown during internal handling

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**